docs/AOT.md

```diff
@@ -62,6 +62,29 @@ This option supports two modes:
 The second generated file is named `<network_name>.weights` and
 contains the weights required to run the compiled model.
 
+Another tool is the `model-compiler` which is used to compile a model into a bundle.
+This tool is more generic (is not tied just to image classification applications)
```

Comment:
Question: May be `image-classifier` should be extended to support this new functionality to avoid having two different tools? We could also create a name alias called `model-compiler` for the existing `image-classifier` as it would support more than just image classification.

Reply:
I think the final (cleanest) solution would be to have 2 tools:
- generic ModelCompiler - compiles any model
- generic ModelRunner - runs any model
I think the image-classifier should be heavily modified to be decoupled from image-classification and even with this modification will not be generic. E.g. a model with multiple inputs, in order to run, should be provided with extra stuff:
- data source for each input (file, stream, etc)
- data decoding method for each input (e.g. encode image/video -> raw, none)
- pre-processing method for each input (normalize [0,1], tokenize)
I think for now we should have them separately until the point when image-classifier will become the ModelRunner I mentioned.

Reply:
Yes, let's keep them separate for now.